pkl-core/src/test/kotlin/org/pkl/core/LanguageSnippetTestsEngine.kt

```diff
@@ -79,6 +79,9 @@ abstract class AbstractLanguageSnippetTestsEngine : InputOutputTestEngine() {
   // can't think of a better solution right now
   protected fun String.stripVersionCheckErrorMessage() =
     replace("Pkl version is ${Release.current().version()}", "Pkl version is xxx")
+  
+  protected fun String.stripStdlibLocationSha(): String =
+    replace(Regex("https://github.com/apple/pkl/blob/[0-9a-fA-F]+/stdlib/"), "https://github.com/apple/pkl/blob/_hash_/stdlib/")
```

Comment:
I think we can match the commit ID directly

```suggestion
    replace("https://github.com/apple/pkl/blob/${Release.current().commitId()}/stdlib/", "https://github.com/apple/pkl/blob/_hash_/stdlib/")
```

Reply:
Indeed. This way we don't need a regex.